src/common/form/select-plain-object.js

```diff
@@ -0,0 +1,113 @@
+import find from 'lodash/find'
+import map from 'lodash/map'
+import React, { Component } from 'react'
+import { Select } from 'form'
+import { propTypes } from 'utils'
+
+@propTypes({
+  autoFocus: propTypes.bool,
+  defaultValue: propTypes.any,
+  disabled: propTypes.bool,
+  multi: propTypes.bool,
+  onChange: propTypes.func,
+  options: propTypes.array,
+  placeholder: propTypes.string,
+  predicate: propTypes.func,
+  required: propTypes.bool
+})
+export class SelectPlainObject extends Component {
```

Comment:
form/select handle options with string/number values like react-select. This one handles any type of values. I made it for my need to handle object values (for the onChange parameter, etc...)

Am I wrong somewhere ?...